### PR TITLE
Setup CI/CD for this upstream repo with Github Action.

### PR DESCRIPTION
**Pull request by HanFa**
This PR setups the Github Action pipeline including
1. lint
2. unit tests
3. integration tests

The stage for building docker image is not included since the internal image registry cannot be accessed.

### CODE REVIEW

**Review by XudongLiuHarold — COMMENTED** [review]
BTW, could you also remove the gitlab ci, since we won't use it anymore. Thanks!

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: Load Balancer Operator for Kubernetes Checks
```
[the remark this answers is not in the document]
Comment (HanFa) [reply .github/workflows/actions.yml:1]: Yes it is possible. We can then setup their dependencies as well. Will do the change.

**Comment by HanFa** [comment]
@XudongLiuHarold there are some changes regarding your comments, could you review this again? Thanks  

1. split the workflow into multiple jobs
2. rm .gitlab-ci

**Review by XudongLiuHarold — APPROVED** [review]
LGTM! Good job @HanFa !

**Pull request merged**